### PR TITLE
dua: add page

### PR DESCRIPTION
**Pull request by thamaraiselvam**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/dua.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`dua aggregate`
+
+- Launch the terminal user interface:
+
+`dua interactive`
+
+- Format printing byte counts:
+
+`dua --format {{format}}`
```
Comment (navarroaxel) [inline pages/common/dua.md:28]: The help says this is all the possible options and case insensitive:
```suggestion
`dua --format {{metric|binary|bytes|GB|GiB|MB|MiB}}`
```

File: pages/common/dua.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Control number of threads to be used:
+
+`dua --threads {{threads-count}}`
```
Comment (navarroaxel) [inline pages/common/dua.md:32]: just `count` is ok
```suggestion
`dua --threads {{count}}`
```

File: pages/common/dua.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Launch the terminal user interface:
+
+`dua interactive`
+
+- Format printing byte counts:
+
+`dua --format {{format}}`
+
+- Control number of threads to be used:
```
Comment (navarroaxel) [inline pages/common/dua.md:30]:
```suggestion
- Set the number of threads to be used:
```

File: pages/common/dua.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# dua
+
+> Dua (Disk Usage Analyzer) is a tool to conveniently learn about the usage of disk space of a given directory.
+> More information: <https://github.com/Byron/dua-cli>.
+
+- Analyze specific directory:
+
+`dua {{directory}}`
```
Comment (navarroaxel) [inline pages/common/dua.md:8]: we use this format for paths:
```suggestion
`dua {{path/to/directory}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @thamaraiselvam, nice command! I left some comments

**Comment by thamaraiselvam** [comment]
@navarroaxel Updated changes

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @thamaraiselvam :slightly_smiling_face:

**Pull request merged**